.github/workflows/build.yml

```
@@ -0,0 +1,406 @@
+    runs-on: ubuntu-latest
+    outputs:
+      release_id: ${{ steps.create-release.outputs.id }}
+      release_upload_url: ${{ steps.create-release.outputs.upload_url }}
+      version: ${{ steps.get-version.outputs.version }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Get version
+        id: get-version
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:8:28: Double quote to prevent globbing and word splitting

(shellcheck)